content/md/user-management/troubleshoot-the-sso.md

```diff
@@ -0,0 +1,58 @@
+---
+id: troubleshoot-the-sso
+themes: user-management
+title: How to **troubleshoot** the Single Sign-On
+popular: false
+ee-only: true
+related: configure-saml-sso
+priority: low
+---
+
+# Overview
+
+Single Sign-On is a powerful tool for productivity but sometimes, as multiple applications are concerned, it can be a hassle to troubleshoot when something goes wrong with it.
+
+Here are a few tips to help you understand what is going on between your `IdP` and your `SP`.
+
+# Make sure the IdP configuration is OK
```

Comment:
there is many "make sure" in this article, you could call your titles "Check your IdP configuration"

Reply:
Yep. I changed the titles with "Check your IdP configuration" and "Make sur the PIM configuration is right"